tutorials/elixir-phoenix-on-google-app-engine.md

```diff
@@ -0,0 +1,185 @@
+    to create a new Cloud Platform project.
+
+2.  Enable billing for your project.
+
+3.  Install the [Google Cloud SDK](https://cloud.google.com/sdk/). Make sure
+    you [initialize](https://cloud.google.com/sdk/docs/initializing) the SDK
+    and set the default project to the new project you created.
+
+    Version 175.0.0 or later of the SDK is required.
+
+If you have not yet installed Elixir and Phoenix, do so:
+
+1.  Install Elixir and NodeJS. If you are on MacOS with Homebrew, you can run
+
+        brew install node elixir
+
+    Otherwise consult the [Node download](https://nodejs.org/en/download/) and
+    [Elixir install](https://elixir-lang.org/install.html) guides for your
+    operating system.
+
+2. Install the hex and phx_new archives.
+
+        mix local.hex
+        mix archive.install https://github.com/phoenixframework/archives/raw/master/phx_new.ez
+
+## Create a new app and run it locally
+
+In this section, you will create a new Phoenix app and make sure it runs. If
+you already have an app to deploy, you may use it instead.
+
+1.  Run the `phx.new` task to create a new Phoenix project called
+    "appengine_example". This tutorial will omit Ecto for now.
+
+        mix phx.new appengine_example --no-ecto
+
+    Answer "Yes" when the tool asks you if you want to fetch and install
+    dependencies.
+
+2.  Go into the directory with the new application.
+
+        cd appengine_example
+
+3.  Run the app with the following command:
+
+        mix phx.server
+
+    This compiles your server and runs it on port 4000.
+
+4.  Visit [http://localhost:4000](http://localhost:4000) to see the Phoenix
+    welcome screen running locally on your workstation.
+
+## Deploy your application
+
+Now you will deploy your new app to App Engine.
+
+1.  Create a file called `app.yaml` at the root of the application directory,
+    with the following contents:
+
+        env: flex
+        runtime: gs://elixir-runtime/elixir.yaml
+
+    This configuration selects the Elixir Runtime, an open source App Engine
+    runtime that knows how to build Elixir and Phoenix applications. You can
+    find more information about this runtime at its
+    [Github page](https://github.com/GoogleCloudPlatform/elixir-runtime).
+
+2.  Run the following command to deploy your app:
+
+        gcloud app deploy
+
+    If this is the first time you have deployed to App Engine in this project,
+    gcloud will prompt you for a region. You may choose the default of
+    "us-central", or select a region close to your location.
+
+    The Elixir Runtime will take care of building your application in the
+    cloud, including installing mix dependencies, compiling to BEAM files, and
+    even using brunch to build and digest your assets.
+
+    Deployment will also take a few minutes to requisition and configure the
+    needed resources, especially the first time you deploy.
+
+3.  Once the deploy command has completed, you can visit
```

Comment:
I like to just say run `gcloud app browse` for this :)

Reply:
done